libs/design/tabs/src/tabs/tab-panel/tab-panel.component.spec.ts

```diff
@@ -80,14 +80,14 @@ describe('@daffodil/design/tabs | DaffTabPanelComponent', () => {
   });
 
   it('should assign the `tabPanelId` value to the `id` attribute', () => {
-    expect(de.attributes['id']).toBe(component.tabPanelId);
+    expect(de.attributes.id).toBe(component.tabPanelId);
   });
 
   it('should set ariaLabelledBy to the tab id', () => {
     expect(component.ariaLabelledBy).toBe(mockTabComponent.id);
   });
 
   it('should set _id to the tab panelId', () => {
-    expect(component['_id']).toBe(mockTabComponent.panelId);
+    expect(component._id).toBe(mockTabComponent.panelId);
```

Comment:
@damienwebdev `_id` is private and can't be accessed in this test. If I change it to use `tabPanelId` since:

```
@HostBinding('attr.id') get tabPanelId() {
    return this._id;
  }
```

it would duplicate the above test:

```
it('should assign the `tabPanelId` value to the `id` attribute', () => {
    expect(de.attributes['id']).toBe(component.tabPanelId);
    expect(de.attributes.id).toBe(component.tabPanelId);
  });
 ```